src/main/java/Duke/Information.java

```diff
@@ -0,0 +1,9 @@
+package Duke;
+
+import Duke.Task.Task;
+
+import java.util.ArrayList;
+
+public class Information {
+    public ArrayList<Task> List = new ArrayList<>();
```

Comment:
Consider using a more descriptive name and allow differentiation between single-valued and multi-valued variables.